CMakeLists.txt

```diff
@@ -282,7 +282,7 @@ IF ( NOT MAKE_DOCS_ONLY )
         CONFIGURE_FILE(${HPCC_WSSQL_SOURCE_DIR}/cmake_modules/dependencies/${packageRevision}.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/dependencies/${packageRevision}.cmake)
         INCLUDE(${CMAKE_CURRENT_BINARY_DIR}/dependencies/${packageRevision}.cmake)
     ELSE()
-        MESSAGE("-- WARNING: DEPENDENCY FILE FOR ${packageRevision} NOT FOUND, Using deps template.")
```

Comment:
It's going to throw a warning now, which is much more visible.

example:
{code}
CMake Warning at cmake_modules/commonSetup.cmake:589 (message):
  USE_OPTIONAL set - missing dependencies for optional features will
  automatically disable them
Call Stack (most recent call first):
  CMakeLists.txt:142 (include)

{code}